src/accessories/RecordingSwitch.ts

```diff
@@ -30,12 +30,11 @@ export class RecordingSwitch extends BaseSwitch {
    */
   async getState(callback: CharacteristicGetCallback) {
     try {
-      const response = await this.logiService.request(
-        'get',
-        `accessories/${this.switchConfig.deviceId}`,
+      const response = await this.logiService.getAccessoryInfo(
+        this.switchConfig.deviceId,
       )
 
-      const state = !response.data.configuration[this.apiPropName]
+      const state = !response.configuration.privacyMode
```

Comment:
The other special ones kept using apiPropName, either revert this one or change the others